packages/contracts-bsc/contracts/BscTrueUSD.sol

```diff
@@ -12,6 +12,15 @@ contract BscTrueUSD is TrueCurrencyWithProofOfReserve {
     uint8 constant DECIMALS = 18;
     uint8 constant ROUNDING = 2;
 
+    function initialize() public {
+        require(!initialized, "already initialized");
+        initialized = true;
+        owner = msg.sender;
+        emit OwnershipTransferred(address(0), owner);
+        burnMin = 1_000_000_000_000_000_000_000; // 1 K
+        burnMax = 1_000_000_000_000_000_000_000_000_000; // 1 B
```

Comment:
After you've granted ownership to `msg.sender`, you can call `setBurnBounds(_burnMin, _burnMax)` in `initialize()` instead of duplicating the function body here.

May involve changing function visibility from `external` to `public`.